spec/live/ticket_spec.rb

```diff
@@ -62,6 +62,23 @@ def valid_attributes
     end
   end
 
+  describe "#show_many" do
+    let(:how_many_tickets) { 10 }
+    before(:all) do
```

Comment:
`before all ` is a bit of a code smell that tends to create order dependent specs. there is only one spec, just `before` which defaults to `:each` should suffice.